### PR TITLE
Added migration for blocking pac bio duplicates creation

### PR DESCRIPTION
**Pull request by emrojo**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

**Comment by emrojo** [comment]
Plan:
- [ ] Deploy SS branch dpl269_duplicate_pacbio_runs
- [ ] Deploy mlwh dpl269_duplicate_pacbio_runs
- [ ] Get the list of IDs that have a duplicate (Run C in sql)
- [ ] In mlwh, rename all pacbio runs from C (175) adding a 'MIGRATED' tag (so it won't be recognised) Run it from a rake task (TODO)
- [ ] In SS, resend all pacbio runs messages from C(175) (TODO)
- [ ] In mlwh, run migrations to create indexes

**Comment by emrojo** [comment]
Test in local:
- [ ] uwh run migration
- [ ] Start SS using training db
- [ ] Start uwh using proddata and listen to local queues
- [ ] Messenger.where(template: 'PacBioRunIO', target_id: list).each(&:broadcast)

**Comment by emrojo** [comment]
And also this pull request: https://github.com/sanger/sequencescape/pull/3555

File: spec/support/it_behaves_like_a_nested_resource.rb [hunk 1]
```diff
@@ -38,7 +38,12 @@ def current_records
 
       before(:each) do
         described_class.create_or_update_from_json(timestamped_json.merge('updated_at' => modified_at), example_lims)
-        described_class.create_or_update_from_json(timestamped_json.merge('updated_at' => modified_at), second_lims)
+        if defined?(json2)
```
Comment (JamesGlover) [inline spec/support/it_behaves_like_a_nested_resource.rb:41]: This should probably just be a different context. in pac_bio_run_spec

**Review by JamesGlover — APPROVED** [review]

**Pull request merged**